.emacs.d/lisp/system/darwin.el

```diff
@@ -0,0 +1,44 @@
+(defun kotct/darwin-swap-command-option ()
+  "This function swaps the Command and Option modifiers on MacOS.
+
+This is intended to make the keyboard layout more friendly for
+those used to a PC-like keyboard layout.  This makes C- and M-
+more spread apart and makes the keyboard more ergonomic.
+
+Because it uses the `mac-command-modifier' and
+`mac-option-modifier' builtins, it does not affect other systems
+built without compatibility for these two variables."
+  (progn
```

Comment:
Can we `defvar` some variable that controls whether or not we do this or not?
Then call these commands after init (on a hook) to make sure that gets used?